typescript/sdk/src/router/HyperlaneRouterDeployer.ts

```diff
@@ -80,17 +79,13 @@ export abstract class HyperlaneRouterDeployer<
 
       await super.runIfOwner(chain, this.router(contracts), async () => {
         const chains = domains.map((id) => this.multiProvider.getChainName(id));
-        this.logger.debug(
+        this.logger.info(
           `Enrolling remote routers (${chains.join(', ')}) on ${chain}`,
         );
         const router = this.router(contracts);
-        const estimatedGas = await router.estimateGas.enrollRemoteRouters(
-          domains,
-          addresses,
-        );
```

Comment:
why remove this?

Reply:
Great catch! In the beginning, when our focus was only on zksync, gas estimation was causing errors and we had to remove it . Will bring it back for sure, here and in other places.